open-source.php

```diff
@@ -155,6 +155,38 @@
     </div>
 </div>
 
+<div class="grid">
+    <div class="whole">
+        <div class="two-thirds">
+            <h2>Projects We've Supported</h2>
```

Comment:
Yeah I was thinking about that as well. I think I personally would rather lean towards the more conservative language even if it's less generous to us just because I feel like it's more honest since some of them are one-offs. I don't want to mislead anyone into thinking that we continually pass money to these projects every month if we've only done it once or twice